libraries/aws_db_parameter_group.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsDbParameterGroup < AwsResourceBase
+  name 'aws_db_parameter_group'
+  desc 'Verifies settings for a Parameter Group'
+  example "
+    describe aws_db_parameter_group(db_parameter_group_name: 'parameter-group') do
```

Comment:
Could you please add another example with out the named arguments

Reply:
Done